--- a/browser/locales/en-US/browser/preferences/preferences.ftl
+++ b/browser/locales/en-US/browser/preferences/preferences.ftl
@@ -1443,14 +1443,14 @@ save-files-to-cloud-storage =
 ## LibreWolf preferences
 
 # Sidebar
-pane-librewolf-title = LibreWolf
+pane-librewolf-title = Yikwid
 category-librewolf =
     .tooltiptext = about:config changes, logically grouped and easily accessible
 
 # Main content
-librewolf-header = LibreWolf Preferences
+librewolf-header = Yikwid Preferences
 librewolf-warning-title = Heads up!
-librewolf-warning-description = We carefully choose default settings to focus on privacy and security. When changing these settings, read the descriptions to understand the implications of those changes.
+librewolf-warning-description = When changing these settings, read the descriptions to understand the implications of those changes.
 
 # Page Layout
 librewolf-general-heading = Browser Behavior
--- a/browser/themes/shared/jar.inc.mn
+++ b/browser/themes/shared/jar.inc.mn
@@ -108,7 +108,7 @@
   skin/classic/browser/preferences/android-menu.svg            (../shared/preferences/android-menu.svg)
   skin/classic/browser/preferences/category-general.svg        (../shared/preferences/category-general.svg)
   skin/classic/browser/preferences/category-privacy-security.svg    (../shared/preferences/category-privacy-security.svg)
-  skin/classic/browser/preferences/category-librewolf.svg    (../shared/preferences/category-librewolf.svg)
+  skin/classic/browser/preferences/category-yikwid.png       (../shared/preferences/category-yikwid.png)
   skin/classic/browser/preferences/category-search.svg         (../shared/preferences/category-search.svg)
   skin/classic/browser/preferences/category-sync.svg           (../shared/preferences/category-sync.svg)
   skin/classic/browser/preferences/containers.css              (../shared/preferences/containers.css)
