--- a/browser/components/preferences/main.inc.xhtml
+++ b/browser/components/preferences/main.inc.xhtml
@@ -29,7 +29,6 @@
   <vbox id="startupPageBox">
     <checkbox id="browserRestoreSession"
               data-l10n-id="startup-restore-windows-and-tabs"/>
-    <html:a is="moz-support-link" support-page="session-restore" data-l10n-id="session-restore-learn-more" />
   </vbox>
 
 #ifdef HAVE_SHELL_SERVICE
