diff --git a/browser/base/content/nsContextMenu.js b/browser/base/content/nsContextMenu.js
index 8397d36..0f69333 100644
--- a/browser/base/content/nsContextMenu.js
+++ b/browser/base/content/nsContextMenu.js
@@ -341,10 +341,13 @@ class nsContextMenu {
     this.initMediaPlayerItems();
     this.initLeaveDOMFullScreenItems();
     this.initClickToPlayItems();
-    this.initPasswordManagerItems();
+    // Yikwid
+    //this.initPasswordManagerItems();
     this.initViewSourceItems();
     this.initScreenshotItem();
-    this.initPasswordControlItems();
+    // Yikwid
+    //this.initPasswordControlItems();
+    document.getElementById("context-reveal-password").hidden = true;
     this.initPDFItems();
 
     this.showHideSeparators(aXulMenu);
@@ -786,7 +789,10 @@ class nsContextMenu {
         !this.onMozExtLink) ||
         this.onPlainTextLink
     );
-    this.showItem("context-keywordfield", this.shouldShowAddKeyword());
+    // Yikwid no search keyword
+    //this.showItem("context-keywordfield", this.shouldShowAddKeyword());
+    document.getElementById("context-keywordfield").hidden = true;
+
     this.showItem("frame", this.inFrame);
 
     if (this.inFrame) {
@@ -2371,7 +2377,8 @@ class nsContextMenu {
     let menuItemPrivate = document.getElementById(
       "context-searchselect-private"
     );
-    if (!Services.search.isInitialized) {
+    // Yikwid always hide search context menu items
+    if (true || !Services.search.isInitialized) {
       menuItem.hidden = true;
       menuItemPrivate.hidden = true;
       return;
