--- a/scripts/librewolf-patches.py
+++ b/scripts/librewolf-patches.py
@@ -58,7 +58,7 @@
 
 def enter_srcdir(_dir = None):
     if _dir == None:
-        dir = "librewolf-{}-{}".format(version, release)
+        dir = "yikwid-{}-{}".format(version, release)
     else:
         dir = _dir
     print("cd {}".format(dir))
@@ -85,6 +85,9 @@
 
 
 def librewolf_patches():
+    exec("rm themes/browser/branding/librewolf/file_librewolf.svg")
+    exec("mv themes/browser/branding/librewolf themes/browser/branding/yikwid")
+    exec("rsync -rv ../themes/browser/branding/yikwid/ themes/browser/branding/yikwid/")
 
     enter_srcdir()
     
@@ -115,9 +116,9 @@
     # Apply most recent `settings` repository files.
     #
 
-    exec('mkdir -p lw')
-    enter_srcdir('lw')
-    exec('cp -v ../../settings/librewolf.cfg .')
+    exec('mkdir -p yw')
+    enter_srcdir('yw')
+    exec('cp -v ../../settings/yikwid.cfg .')
     exec('cp -v ../../settings/distribution/policies.json .')
     exec('cp -v ../../settings/defaults/pref/local-settings.js .')
     leave_srcdir();
@@ -131,7 +132,6 @@
     # 1) patch it in
     patch('../patches/pref-pane/pref-pane-small.patch')
     # 2) new files
-    exec('cp ../patches/pref-pane/category-librewolf.svg browser/themes/shared/preferences/category-librewolf.svg')
     exec('cp ../patches/pref-pane/librewolf.css browser/themes/shared/preferences/librewolf.css')
     exec('cp ../patches/pref-pane/librewolf.inc.xhtml browser/components/preferences/librewolf.inc.xhtml')
     exec('cp ../patches/pref-pane/librewolf.js browser/components/preferences/librewolf.js')
@@ -142,8 +142,8 @@
 
     
     # provide a script that fetches and bootstraps Nightly and some mozconfigs
-    exec('cp -v ../scripts/mozfetch.sh lw/')
-    exec('cp -v ../assets/mozconfig.new ../assets/mozconfig.new.without-bootstrap ../scripts/setup-wasi-linux.sh lw/')
+    exec('cp -v ../scripts/mozfetch.sh yw/')
+    exec('cp -v ../assets/mozconfig.new ../assets/mozconfig.new.without-bootstrap ../scripts/setup-wasi-linux.sh yw/')
 
     # override the firefox version
     for file in ["browser/config/version.txt", "browser/config/version_display.txt"]:
@@ -153,11 +152,11 @@
     with TemporaryDirectory() as tmpdir:
         exec(f"wget -qO {tmpdir}/l10n.zip 'https://codeload.github.com/mozilla-l10n/firefox-l10n/zip/refs/heads/main'")
         exec(f"unzip -qo {tmpdir}/l10n.zip -d {tmpdir}/l10n")
-        exec(f"mv {tmpdir}/l10n/firefox-l10n-main lw/l10n")
+        exec(f"mv {tmpdir}/l10n/firefox-l10n-main yw/l10n")
 
     print("-> Patching appstrings.properties")
     # Why is "Firefox" hardcoded there???
-    exec("find . -path '*/appstrings.properties' -exec sed -i s/Firefox/LibreWolf/ {} \;")
+    exec("find . -path '*/appstrings.properties' -exec sed -i s/Firefox/Yikwid/ {} \;")
 
     print("-> Applying LibreWolf locales")
     l10n_dir = Path("..", "l10n")
@@ -174,7 +174,7 @@
             )
         else:
             target_path = Path(
-                "lw", "l10n",
+                "yw", "l10n",
                 *rel_path.parts[0:2],
                 *rel_path.parts[1:]
             )
@@ -162,11 +162,11 @@
 #
 
 if len(args) != 2:
-    sys.stderr.write('error: please specify version and release of librewolf source')
+    sys.stderr.write('error: please specify version and release of yikwid source')
     sys.exit(1)
 version = args[0]
 release = args[1]
-srcdir = "librewolf-{}-{}".format(version, release)
+srcdir = "yikwid-{}-{}".format(version, release)
 if not os.path.exists(srcdir + '/configure.py'):
     sys.stderr.write('error: folder doesn\'t look like a Firefox folder.')
     sys.exit(1)
