--- a/toolkit/xre/nsXREDirProvider.cpp
+++ b/toolkit/xre/nsXREDirProvider.cpp
@@ -279,7 +279,7 @@ nsresult nsXREDirProvider::GetBackgroundTasksProfilesRootDir(
  *   (for 32- and 64-bit systems respsectively)
  */
 static nsresult GetSystemParentDirectory(nsIFile** aFile,
-                                         nsCString aName = "LibreWolf"_ns) {
+                                         nsCString aName = "Yikwid"_ns) {
   nsresult rv;
   nsCOMPtr<nsIFile> localDir;
 #  if defined(XP_MACOSX)
@@ -370,9 +370,9 @@ nsXREDirProvider::GetFile(const char* aProperty, bool* aPersistent,
     rv = GetUserDataDirectoryHome(getter_AddRefs(file), false);
     NS_ENSURE_SUCCESS(rv, rv);
 #  if defined(XP_MACOSX)
-    rv = file->AppendNative("LibreWolf"_ns);
+    rv = file->AppendNative("Yikwid"_ns);
 #  else   // defined(XP_MACOSX)
-    rv = file->AppendNative(".librewolf"_ns);
+    rv = file->AppendNative(".yikwid"_ns);
 #  endif  // defined(XP_MACOSX)
   } else if (!strcmp(aProperty, XRE_MOZ_USER_NATIVE_MANIFESTS)) {
     rv = GetUserDataDirectoryHome(getter_AddRefs(file), false);
@@ -411,9 +411,9 @@ nsXREDirProvider::GetFile(const char* aProperty, bool* aPersistent,
 #  ifdef ENABLE_SYSTEM_EXTENSION_DIRS
 #    if defined(__OpenBSD__) || defined(__FreeBSD__)
     static const char* const sysLExtDir =
-        "/usr/local/share/librewolf/extensions";
+        "/usr/local/share/yikwid/extensions";
 #    else
-    static const char* const sysLExtDir = "/usr/share/librewolf/extensions";
+    static const char* const sysLExtDir = "/usr/share/yikwid/extensions";
 #    endif
     rv = NS_NewNativeLocalFile(nsDependentCString(sysLExtDir),
                                getter_AddRefs(file));
@@ -939,7 +939,7 @@ nsresult nsXREDirProvider::GetUpdateRootDir(nsIFile** aResult,
   }
   appDirPath = Substring(appDirPath, 1, dotIndex - 1);
 
-  if (NS_FAILED(localDir->AppendNative("LibreWolf"_ns))) {
+  if (NS_FAILED(localDir->AppendNative("Yikwid"_ns))) {
     return NS_ERROR_FAILURE;
   }
 
@@ -1199,7 +1199,7 @@ nsresult nsXREDirProvider::AppendSysUserExtensionPath(nsIFile* aFile) {
 
 #if defined(XP_MACOSX) || defined(XP_WIN)
 
-  static const char* const sXR = "LibreWolf";
+  static const char* const sXR = "Yikwid";
   rv = aFile->AppendNative(nsDependentCString(sXR));
   NS_ENSURE_SUCCESS(rv, rv);
 
@@ -1209,7 +1209,7 @@ nsresult nsXREDirProvider::AppendSysUserExtensionPath(nsIFile* aFile) {
 
 #elif defined(XP_UNIX)
 
-  static const char* const sXR = ".librewolf";
+  static const char* const sXR = ".yikwid";
   rv = aFile->AppendNative(nsDependentCString(sXR));
   NS_ENSURE_SUCCESS(rv, rv);
 
