--- a/toolkit/components/search/SearchService.sys.mjs
+++ b/toolkit/components/search/SearchService.sys.mjs
@@ -1219,6 +1219,9 @@ export class SearchService {
    *   The appropriate search engine, or null if one could not be determined.
    */
   _getEngineDefault(privateMode) {
+    // Yikwid no search engines
+    return null;
+
     let currentEngine = privateMode
       ? this.#currentPrivateEngine
       : this.#currentEngine;
@@ -1271,6 +1274,9 @@ export class SearchService {
    * Throws in case of initialization error.
    */
   #ensureInitialized() {
+    // Yikwid no search service
+    return;
+
     if (this.#initializationStatus === "success") {
       return;
     }
@@ -1572,6 +1572,9 @@ export class SearchService {
    *   The refined search configuration for this user.
    */
   async #loadEngines(settings, refinedConfig) {
+    // Yikwid no engine loading
+    return;
+
     // Get user's current settings and search engine before we load engines from
     // config. These values will be compared after engines are loaded.
     let prevMetaData = { ...settings?.metaData };
@@ -1883,6 +1892,9 @@ export class SearchService {
   // This is prefixed with _ rather than # because it is called in
   // test_remove_engine_notification_box.js
   async _reloadEngines(settings, changeReason) {
+    // Yikwid no loading engines
+    return;
+
     // Capture the current engine state, in case we need to notify below.
     let prevCurrentEngine = this.#currentEngine;
     let prevPrivateEngine = this.#currentPrivateEngine;
@@ -2864,6 +2876,9 @@ export class SearchService {
    *   The appropriate search engine, or null if one could not be determined.
    */
   #findAndSetNewDefaultEngine({ privateMode }, changeReason) {
+    // Yikwid no engines
+    return null;
+
     // First to the app default engine...
     let newDefault = privateMode
       ? this.appPrivateDefaultEngine
