--- a/browser/base/content/nsContextMenu.sys.mjs
+++ b/browser/base/content/nsContextMenu.sys.mjs
@@ -350,10 +350,13 @@ export class nsContextMenu {
     this.initClipboardItems();
     this.initMediaPlayerItems();
     this.initLeaveDOMFullScreenItems();
-    this.initPasswordManagerItems();
+    // Yikwid
+    //this.initPasswordManagerItems();
     this.initViewSourceItems();
     this.initScreenshotItem();
-    this.initPasswordControlItems();
+    // Yikwid
+    //this.initPasswordControlItems();
+    this.document.getElementById("context-reveal-password").hidden = true;
     this.initPDFItems();
     this.initTextFragmentItems();
 
@@ -900,8 +903,11 @@ export class nsContextMenu {
         !this.onMozExtLink) ||
         this.onPlainTextLink
     );
-    this.showItem("context-add-engine", this.shouldShowAddEngine());
-    this.showItem("context-keywordfield", this.shouldShowAddKeyword());
+    // Yikwid no search keyword
+    //this.showItem("context-add-engine", this.shouldShowAddEngine());
+    //this.showItem("context-keywordfield", this.shouldShowAddKeyword());
+    document.getElementById("context-add-engine").hidden = true;
+    document.getElementById("context-keywordfield").hidden = true;
     this.showItem("frame", this.inFrame);
 
     if (this.inFrame) {
@@ -2880,7 +2886,8 @@ export class nsContextMenu {
     if (!menuitem) {
       return;
     }
-    if (!Services.search.hasSuccessfullyInitialized) {
+    // Yikwid always hide search context menu items
+    if (true || !Services.search.hasSuccessfullyInitialized) {
       menuitem.hidden = true;
       return;
     }
