--- a/browser/extensions/newtab/lib/AboutPreferences.sys.mjs
+++ b/browser/extensions/newtab/lib/AboutPreferences.sys.mjs
@@ -250,6 +250,8 @@ export class AboutPreferences {
       element.disabled = Preferences.get(fullPref).locked;
     };
 
+    // Yikwid: new tab is removed, don't display settings
+    shouldHidePref = true;
     // Don't show any sections that we don't want to expose in preferences UI
     if (shouldHidePref) {
       return;
