--- a/browser/components/preferences/privacy.js
+++ b/browser/components/preferences/privacy.js
@@ -2196,17 +2196,6 @@ Preferences.addSetting({
     return privateBrowsingAutoStart.value;
   },
 });
-Preferences.addSetting({
-  id: "rememberForms",
-  pref: "browser.formfill.enable",
-  deps: ["historyMode", "privateBrowsingAutoStart"],
-  visible({ historyMode }) {
-    return historyMode.value == "custom";
-  },
-  disabled({ privateBrowsingAutoStart }) {
-    return privateBrowsingAutoStart.value;
-  },
-});
 Preferences.addSetting({
   id: "alwaysClear",
   pref: "privacy.sanitize.sanitizeOnShutdown",
