--- a/scripts/generate-locales.sh
+++ b/scripts/generate-locales.sh
@@ -30,15 +30,18 @@
   mv browser/locales/l10n/$1-*/ browser/locales/l10n/$1/
   rm -f browser/locales/l10n/$1.zip
   echo_status "Generating locale \"$1\""
-  mv browser/locales/l10n/$1/browser/branding/official browser/locales/l10n/$1/browser/branding/librewolf
-  find browser/locales/l10n/$1 -type f -exec sed -i -e 's/Mozilla Firefox/LibreWolf/g' {} \;
-  find browser/locales/l10n/$1 -type f -exec sed -i -e 's/Mozilla/LibreWolf/g' {} \;
-  find browser/locales/l10n/$1 -type f -exec sed -i -e 's/Firefox/LibreWolf/g' {} \;
+  mv browser/locales/l10n/$1/browser/branding/official browser/locales/l10n/$1/browser/branding/yikwid
+  find browser/locales/l10n/$1 -type f -exec sed -i -e 's/Mozilla Firefox/Yikwid Browser/g' {} \;
+  find browser/locales/l10n/$1 -type f -exec sed -i -e 's/Mozilla/Yikwid LLC/g' {} \;
+  find browser/locales/l10n/$1 -type f -exec sed -i -e 's/Firefox/Yikwid/g' {} \;
   echo_status "Done"
   sleep 0.3
   echo_status
 }
 
+# Yikwid: only en-US locale
+echo "en-US" > browser/locales/shipped-locales
+
 while read in; do
   ((n=n%N)); ((n++==0)) && wait
   generate_locale $in &
